src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -31,6 +33,21 @@ namespace Microsoft.PowerShell.EditorServices
     /// </summary>
     public class PowerShellContext : IDisposable, IHostSupportsInteractiveSession
     {
+        private const string DotNetFrameworkDescription = ".NET Framework";
```

Comment:
Nit: this is more of a "Name" the a "Description" IMO

Reply:
I considered calling it that, but it's compared to `RuntimeInformation.FrameworkDescription`, which is why I called it `Description`.